sonic-net/sonic-mgmt
# Add test case `test_custom_acl` to verify custom ACL table

### Description of PR


Summary:
The PR is to add a new test case `test_custom_acl` to verify custom ACL table.
Test steps

1. Define a custom ACL table type by loading json configuration
2. Create an ingress ACL table with the custom type
3. Toggle all ports to active if the test is running on dual-tor
4. Ingress packets from vlan port
5. Verify the packets are egressed to uplinks 
6. Verify the counter of expected rule increases as expected

Depends on PR https://github.com/sonic-net/sonic-swss/pull/2550
### Type of change





### Back port request

### Approach
#### What is the motivation for this PR?
The PR is to add a new test case `test_custom_acl` to verify custom ACL table.

#### How did you do it?

#### How did you verify/test it?
Verified on Mellanox platform, running `202205` image.

#### Any platform specific information?
No.
#### Supported testbed topology if it's a new test case?
No.
### Documentation
